### PR TITLE
Wip/hughsie/verbose

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Comment by superm1** [comment]
This LGTM but I wonder about also being able to set domain whitelist from command line? I like how `--plugin-whitelist` can be called with multiple plugins, what about having `--verbose` behave same way?

**Comment by hughsie** [comment]
We can't re-use --verbose without enforcing a string domain -- but we can do something like ^^

**Comment by superm1** [comment]
Yeah, that works pretty well.

I think the domain for ThunderboltPower needs to be shortened or the max length increased though.
`FuPluginThunderboltPower` is much larger

File: src/fu-debug.c [hunk 1]
```diff
@@ -116,6 +118,9 @@ fu_debug_pre_parse_hook (GOptionContext *context,
 		{ "plugin-verbose", '\0', 0, G_OPTION_ARG_STRING_ARRAY, &self->plugin_verbose,
 		  /* TRANSLATORS: this is for plugin development */
 		  N_("Show plugin verbose information"), "PLUGIN-NAME" },
+		{ "daemon-verbose", '\0', 0, G_OPTION_ARG_STRING_ARRAY, &self->daemon_verbose,
+		  /* TRANSLATORS: this is for daemon development */
+		  N_("Show daemon verbose information"), "DOMAIN" },
```
Comment (superm1) [inline src/fu-debug.c]: Maybe:
`Show daemon verbose information for individual domains`

**Comment by superm1** [comment]
Other than my comment about maybe renaming that string, this PR looks good to me now (feel free to squash or modify the patches I added though)

File: src/fu-debug.c (hunk 1 above)
Comment (hughsie) [reply src/fu-debug.c]: Nahh, domain is a term overloaded by multiple things. It needs to be translatable.

Comment (superm1) [reply src/fu-debug.c]: OK

**Pull request merged**